### PR TITLE
Token-level explanations

### PR DESCRIPTION
**Pull request by jppgks**

### CODE REVIEW

**Review by connor-mccorm — APPROVED** [review]

File: ludwig/explain/captum.py [hunk 1]
```diff
@@ -158,51 +176,132 @@ def explain(self) -> Tuple[List[Explanation], List[float]]:
...
     target_idx: Optional[int],
-    inputs_encoded: List[Variable],
-    baseline: List[Variable],
+    feature_inputs: List[Variable],
+    baseline: List[torch.Tensor],
     use_global: bool,
     nsamples: int,
-) -> np.array:
+) -> Tuple[npt.NDArray[np.float64], Dict[str, List[List[Tuple[str, float]]]]]:
+    """Compute the total attribution for each input feature for each row in the input data.
```
Comment (justinxzhao) [inline ludwig/explain/captum.py:229]: nit: Use Google python style docstrings:
https://google.github.io/styleguide/pyguide.html#383-functions-and-methods

File: ludwig/explain/captum.py [hunk 2]
```diff
@@ -221,4 +320,46 @@ def get_total_attribution(
...
+def get_token_attributions(
+    model: LudwigModel,
+    feature_name: str,
+    input_ids: torch.Tensor,
+    token_attributions: torch.Tensor,
+) -> List[List[Tuple[str, float]]]:
+    """Convert token-level attributions to an array of token-attribution pairs of shape [batch_size,
+    sequence_length, 2].
+
+    # Inputs
```
Comment (justinxzhao) [inline ludwig/explain/captum.py:337]: nit: Use https://google.github.io/styleguide/pyguide.html#383-functions-and-methods

File: ludwig/explain/captum.py (hunk 1 above)
Comment (connor-mccorm) [reply ludwig/explain/captum.py:229]: Sounds good, will change now!

File: ludwig/explain/captum.py (hunk 2 above)
Comment (connor-mccorm) [reply ludwig/explain/captum.py:337]: Made the change!

**Review by justinxzhao — APPROVED** [review]

**Pull request merged**